docs/how-to/index.rst

```diff
@@ -33,14 +33,15 @@ Learn how to manage `MLflow <Charmed MLflow_>`_ within your DSS environment.
    :maxdepth: 1
 
    mlflow
-
-Leverage NVIDIA GPUs
+   
+Leverage GPUs
```

Comment:
We need to create a folder in the repo, so this two guides appear within the same dropdown in the left-hand sidebar. We can call it "Leverage GPUs" or "Enable GPUs". Both options are fine with me.

Reply:
Updated.